base/inc/post-selector.php

```diff
@@ -98,6 +99,10 @@ function siteorigin_widget_post_selector_process_query( $query ){
 		}
 	}
 
+	if ( $exclude_current && get_the_ID() ) {
+		$query['post__not_in'][] = get_the_ID();
```

Comment:
Are there situations where we'd need to initialise `$query['post__not_in']` to an empty array if it's not set yet?

Reply:
I wasn't able to come across any. I ran a few tests where `post__not_in` was, and wasn't used and didn't run into any issues/errors/warnings. I can initialize it if you would prefer.

Reply:
Initialising is always slightly safer, so I'd say yes.